src/css/card.css

```
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 200px;
```

```suggestion
  width: 200px;
```
I think it's better to make the width fixed